quinn-proto/src/connection.rs

```diff
@@ -113,6 +121,8 @@ pub struct Connection {
     /// count towards bytes_in_flight to ensure congestion control does not impede congestion
     /// feedback.
     bytes_in_flight: u64,
+    /// Number of packets bearing CRYPTO frames in flight
```

Comment:
Maybe keep this in its former order in this commit and move it in a later commit?